### DISCUSSION TITLE
lua stage back image displays as black

### DISCUSSION THREAD

**Discussion by charlesisfeline** [DEMOLITIONDON96/Demolition-Engine#23] [Lua Q&A]
![image](https://user-images.githubusercontent.com/62410083/162483166-3bea4bc5-346f-453d-be49-07151035b2d8.png)

fix this please

**Comment by ThatOneSillyGuyDon** [comment]
Did you type the image name, directory, x and y offsets correctly?

**Reply by charlesisfeline** [reply]
hold on let me check

**Reply by charlesisfeline** [reply]
i think i did
![image](https://user-images.githubusercontent.com/62410083/162489915-3135f86b-2c69-492d-b6ee-12b663fbdea7.png)

**Reply by charlesisfeline** [reply]
and the lua code's all correct

**Reply by charlesisfeline** [reply]
nah it’s all fine

**Reply by ThatOneSillyGuyDon** [reply]
> i think i did ![image](https://user-images.githubusercontent.com/62410083/162489915-3135f86b-2c69-492d-b6ee-12b663fbdea7.png)

huh, strange, it works fine for me when i try this with my assets, i mean, it SHOULD work just fine, since we didn't modify any of the existing lua functions

**Comment by ghost** [comment]
You can't use normal floats with decimals inside of x and y positions. They need to be Integers. (whole numbers)
Try rounding them.
```lua
angleshit = 1
anglevar = 1

function onCreate()
    makeLuaSprite('bg', 'whitwall', -410, -295)
    setLuaSpriteScrollFactor('bg' 0.9, 0.9)
    addLuaSprite('bg', false)

    makeLuaSprite('bgg', 'whitfloor', -410, -295)
    setLuaSpriteScrollFactor('bgg' 1, 0.95)
    addLuaSprite('bgg', false)
end
```
